core/XH.ts

```diff
@@ -255,7 +255,7 @@ export class XHApi {
     @observable
     appState: AppState = 'PRE_AUTH';
 
-    /** milliseconds since user activity / interaction was last detected. */
+    /** milliseconds timestamp at moment of last user activity / interaction was last detected. */
```

Comment:
To maybe get right to the bigger question, should we consider doing something like making the property on XH observable and update it with a debounced handler?  Would that be a simpler way to make the existing property more useful - an app could e.g. add an `@computed` to report when the diff between then and now crosses their threshold - without the extra complexity of another timer in IdleService?

It seems a bit confusing to have IdleService officially disabled, and yet reporting / maintaining an `idleFor` value